### DISCUSSION TITLE
Person: error when different person with the same name

### DISCUSSION THREAD

**Discussion by dfldylan** [photoprism/photoprism#2399] [Q&A]
I have two people with the same name, but they're going to merge into the same person in the PERSON area.
Similar problems are more serious in Chinese names, where there are many names with the same sound and different characters.
<img width="262" alt="image" src="https://user-images.githubusercontent.com/28103567/171972232-b13b6c3d-119e-4eb6-b1b3-c204247c30b8.png">

**Comment by graciousgrey** [comment]
That's expected. 
If you add a name to a face, that already exists in people, this face gets assigned to the existing person.
We might later (once it is possible to add more information to people --> https://github.com/photoprism/photoprism/issues/1552 add an option to create the same name twice e.g. with different birth dates. 
For now you need to use different names e.g. "Jane Doe 1" "Jane Doe 2"

**Reply by dfldylan** [reply]
Yeah, actually I was wondering if it would be better to use uid as index?

**Reply by graciousgrey** [reply]
There is a uid in the subjects table. So yes in theory there could be subjects with the same name (and we might support it later).
But it would not be convenient to present uids instead of names in the dropdown when users want to add names to faces.
